server/graphql/v1/mutations/paymentMethods.js

```diff
@@ -195,3 +195,28 @@ export async function updatePaymentMethod(args, remoteUser) {
 
   return models.PaymentMethod.update(pick(args, allowedFields), { where: { id: paymentMethod.id } });
 }
+
+/** Update payment method with given args */
+export async function replaceCreditCard(args, remoteUser) {
+  const oldPaymentMethod = await models.PaymentMethod.findByPk(args.id);
+  if (!oldPaymentMethod || !remoteUser || !remoteUser.isAdmin(oldPaymentMethod.CollectiveId)) {
+    throw PaymentMethodPermissionError;
+  }
+
+  args = omit(args, ['id']);
```

Comment:
It's not a problem today, but more as a code style, I would avoid rewriting `args` to a new value.

Reply:
Note: see proposed solution in Slack